groovy/replica-pipeline

```diff
@@ -0,0 +1,39 @@
+#!/bin/groovy
```

Comment:
Throughout this script, it is inconsistent whether there is or isn't a space before an opening curly brace. We should probably have just one standard for the formatting.

Reply:
I agree.. let me adjust it..